### PR TITLE
Optional database and port using environment variable

### PR DESCRIPTION
**Pull request by herpiko**
@dotovr,
DB=namadb PORT=8080 dst...

### CODE REVIEW

File: gulp/tasks/development.js [hunk 1]
```diff
@@ -9,6 +9,6 @@ gulp.task('dev', ['clean'], function(cb) {
 
   global.isProd = false;
 
-  runSequence(['styles', 'images', 'browserExtensions', 'fonts', 'views', 'browserify'], 'watch', cb);
+  runSequence(['db','styles', 'images', 'browserExtensions', 'fonts', 'views', 'browserify'], 'watch', cb);
```
Comment (mdamt) [inline gulp/tasks/development.js]: watch ganti gzip
browserExtensions dihapus

**Comment by herpiko** [comment]
@mdamt Sudah masuk.

**Comment by emhaye** [comment]
env DB belum bisa

**Comment by emhaye** [comment]
screenshot from 2015-10-21 13-39-01

**Comment by emhaye** [comment]
:+1:

**Comment by emhaye** [comment]
:+1:

**Pull request merged**